tests/test_make.py

```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Fair enough. I don't think travis keeps any output files, though, so we would need to `make test` (or whatever) manually to look at the compiled pdfs.